tests/x11/gnome_music.pm

```diff
@@ -17,7 +17,12 @@ use testapi;
 use utils;
 
 sub run {
-    assert_gui_app('gnome-music');
+    assert_gui_app('gnome-music', install => 1);
+    send_key('alt-f4');
+}
+
+sub test_flags {
+    return {milestone => 1};
```

Comment:
why should it be milestone?

Reply:
I meant that failure of this test should not lead to failure of whole test suite. At the same time update of "lastgood" can be handy during debugging in this particular case.

Reply:
Hm, are you aware about http://open.qa/docs/#_how_to_write_tests ? A failure in one module will by default not make the scenario fail. And "milestone" should only be used if the test module changes something important to the system to update the point to rollback to in case of later test modules failing. I do not think this applies for gnome_music. And "handy during debugging" would apply to basically all test modules, doesn't it? :) For debugging keep in mind there is http://open.qa/docs/#_using_snapshots_to_speed_up_development_of_tests which is basically doing the same as setting the "milestone"-flag on *all* test modules temporarily.